com.github.kevinsmia1939.PySimpleCV.yaml

```diff
@@ -0,0 +1,98 @@
+app-id: com.github.kevinsmia1939.PySimpleCV
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+command: PySimpleCV
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11 #PySimpleGUI Wayland not support
+  - --device=dri
+  - --filesystem=home # No protal support
+modules:
+  - name: python3-numpy
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/42/38/775b43da55fa7473015eddc9a819571517d9a271a9f8134f68fb9be2f212/numpy-1.23.5.tar.gz
+        sha256: 1b1766d6f397c18153d40015ddfc79ddb715cabadc04d2d228d4e5a8bc4ded1a
+        x-checker-data:
+          type: pypi
+          name: numpy
+
+  - name: pybind11
+    buildsystem: simple
+    build-commands:
+      - python3 -mpip install . --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST}
```

Comment:
are you sure? It should be either this or a `cmake-ninja` buildsystem. not both.